app/src/main/kotlin/compiler/intermediate/generators/FunctionDetailsGenerator.kt

```diff
@@ -7,20 +7,20 @@ import compiler.intermediate.IFTNode
 interface FunctionDetailsGenerator : VariableAccessGenerator {
     data class FunctionCallIntermediateForm(
         val callGraph: ControlFlowGraph,
-        val result: IFTNode?
+        val result: IFTNode?,
+        val stateForResumingGenerator: IFTNode?
```

Comment:
This can actually be general-purpose (SysV calling convention allows using both RAX and RDX for return values), that's why I proposed to simply name it `secondResult`. Currently we are only planning to use it for generator functions, so I guess it's not an issue for now.